docs/src/main/tut/numbers/algebraic.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: docs
+title:  "Algebraic"
+section: "numbers"
+source: "core/shared/src/main/scala/spire/math/Algebraic.scala"
+scaladoc: "#spire.math.Algebraic"
```

Comment:
I cribbed this style from Cats - as far as I can tell, neither they nor the algebird microsite are using this info now, but I think it'd be great to have a custom Jekyll layout that added links to these items.